server/stycoon/ships.go

```diff
@@ -0,0 +1,28 @@
+package stycoon
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func (game *Game) SetShipClasses() error {
+	shipClasses := make(map[string]StaticDataShipClassesValue)
+	rows, err := game.db.Query("select * from d_class")
+	if err != nil {
+		return fmt.Errorf("query failed %v", err)
+	}
+	var shipClass StaticDataShipClassesValue
+	var id int
+	for rows.Next() {
+		err = rows.Scan(&id, &shipClass.Name, &shipClass.Shipyard, &shipClass.Speed, &shipClass.CargoCapacity, &shipClass.Life, &shipClass.Damage, &shipClass.Price)
```

Comment:
does this use the variable names to map to the correct columns from the database? or is it just based on the order of the variables?
this table has been recently changed in the database and few new columns were added.

Reply:
just the order, added explicit listing